Header header
string[] joint_names 
float64[] positions
float64[] velocities
float64[] accelerations
float64[] max_currents